### ISSUE TITLE
How to divide validation set?

### ISSUE THREAD

**Issue by jwang-rs** [HopooLinZ/DAOT#4]
When retraining and finetuning A2B, is it proper to use ShanghaiB_test imgs and its gt label as validation set directly, or is there any other way to validate best checkpoint?

**Comment by HopooLinZ** [comment]
Thank you for your suggestion. In previous experiments, we directly used ShanghaiB_test images for validation. We appreciate your input and will consider using pseudo-labels to validate the best checkpoint in our future work.
